client/src/components/Input/FormInput.js

```diff
@@ -1,36 +1,75 @@
 import React from "react";
-import { theme } from "constants/theme";
+import styled from "styled-components";
+import { mq } from "constants/theme";
+import SvgIcon from "../Icon/SvgIcon";
+const FormInput = styled.input`
+  border-top-style: hidden;
+  border-left-style: hidden;
+  border-right-style: hidden;
+  border-color: ${(props) => (props.error ? "#FF5656" : "#5970EC")};
+  border-width: thin;
+  margin-bottom: 2rem;
+  margin-top: 0.4rem;
+  padding-bottom: 0.5rem;
+  color: #000000;
+  @media screen and (min-width: ${mq.tablet.narrow.minWidth}) {
+    margin-top: 1rem;
+    margin-bottom: 4rem;
+  }
+  :last-of-type {
+    margin-bottom: 2rem;
+  }
+`;
 
-const { black, orangeRed, primary, royalBlue } = theme.colors;
+const Icon = styled(SvgIcon)`
+  display: none;
+  @media screen and (min-width: ${mq.tablet.narrow.minWidth}) {
+    display: initial;
+    margin-right: 1rem;
+  }
+`;
 
+const IconWrapper = styled.div`
+  display: flex;
+  align-items: baseline;
+`;
+// const inputFieldStyle = {
```

Comment:
can you remove this or will this still be usable?

Reply:
removed